### PR TITLE
Improve AuthorizationCode docs

### PR DESCRIPTION
**Pull request by YLSDev**

### CODE REVIEW

File: EpicGames/Web/Id/Auth/AuthorizationCode.md [hunk 1]
```diff
@@ -6,17 +6,24 @@ Auth Required: Yes
 
 ## Query Parameters
 
-`clientId`: The Id of the Client that the Authorization Code should be issued for <br/>
-`responseType`: Must be `code` <br/>
+`clientId`: The ID of the Client that the Authorization Code should be issued for <br/>
+`responseType`: The responseType you want to use (for non-EOS Clients: `code`, `eula_token` for EOS Clients: `code`) <br/>
```
Comment (Jaren8r) [inline EpicGames/Web/Id/Auth/AuthorizationCode.md:10]: Considering we are documenting more than just how to get an authorization code from the redirect endpoint (and considering the redirect endpoint can now generate exchange codes which has yet to be documented), I feel like we should rename the whole file to Redirect.md

File: EpicGames/Web/Id/Auth/AuthorizationCode.md [hunk 2]
```diff
@@ -6,17 +6,24 @@ Auth Required: Yes
...
 `exchangeCode`: (optional), a valid exchange code (the usage is unknown) <br/>
 `codeChallenge`: (optional), a PKCE code challenge <br/>
 `codeChallengeMethod`: (optional), the method used to generate the code challenge (`S256` or `plain`) <br/>
-`scope`: (optional), a space seperated list of scopes <br/>
-`redirectUrl`: (optional), Automaticly redirect to that URL with the `code` Query being set to the created Authorization Code
+`nonce`: (optional) A valid [OpenID Nonce](https://curity.io/resources/learn/nonce-authenticator-pattern/#:~:text=The%20nonce%20authenticator%20pattern%20enables,session%20in%20the%20target%20application.&usg=AOvVaw2AvRUGFE9xV6e0C4-ifp_P&opi=89978449) <br/>
+`scope`: (optional), A space seperated list of scopes <br/>
+`eula`: (optional, deprecated), A valid eula ID (ex. `fn`, `tos`) <br/>
+`prompt`: (optional), The redirect prompt, (ex. `none`, `login`, `new_session`, `register`, `consent`, `pass_through`, `2nd_domain`, `upgrade`)  <br/>
+`redirectUrl`: (optional), The URL to redirect with the `code` Param being set to the created Authorization Code
```
Comment (Jaren8r) [inline EpicGames/Web/Id/Auth/AuthorizationCode.md:22]:
```suggestion
`redirectUrl`: (optional), The URL to redirect to
```

Query parameters being set on the url depends on the responseType, which more than one of is documented now.

**Comment by LeleDerGrasshalmi** [comment]
@Jaren8r does it look good now?

File: EpicGames/Web/Id/Auth/Redirect.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+## Id Web - Create Authorization Code
```
Comment (Jaren8r) [inline EpicGames/Web/Id/Auth/Redirect.md:1]:
```suggestion
## Id Web - Redirect
```

File: EpicGames/Web/Id/Auth/Redirect.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+`userCode`: (optional) A valid [User Code](../../../AccountService/Authentication/DeviceCode/Create.md) <br/>
+`continuation`: (optional) A valid Continuation Token <br/>
+`exchangeCode`: (optional), a valid exchange code (the usage is unknown) <br/>
+`codeChallenge`: (optional), a PKCE code challenge <br/>
+`codeChallengeMethod`: (optional), the method used to generate the code challenge (`S256` or `plain`) <br/>
+`nonce`: (optional) A valid [OpenID Nonce](https://curity.io/resources/learn/nonce-authenticator-pattern/#:~:text=The%20nonce%20authenticator%20pattern%20enables,session%20in%20the%20target%20application.&usg=AOvVaw2AvRUGFE9xV6e0C4-ifp_P&opi=89978449) <br/>
+`scope`: (optional), A space seperated list of scopes <br/>
+`eula`: (optional, deprecated), A valid eula ID (ex. `fn`, `tos`) <br/>
+`prompt`: (optional), The redirect prompt, (ex. `none`, `login`, `new_session`, `register`, `consent`, `pass_through`, `2nd_domain`, `upgrade`)  <br/>
+`redirectUrl`: (optional), The URL to redirect t
```
Comment (Jaren8r) [inline EpicGames/Web/Id/Auth/Redirect.md:22]:
```suggestion
`redirectUrl`: (optional), The URL to redirect to
```

File: EpicGames/Web/Id/Auth/Redirect.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+`clientId`: The ID of the Client that the Authorization Code should be issued for <br/>
+`responseType`: The responseType you want to use (for non-EOS Clients: `code`, `eula_token` for EOS Clients: `code`) <br/>
+`provider`: (optional) The external account provider (ex. psn, nintendo, ps5, xbox, vk, apple, lego) <br/>
+`state`: (optional) The login state obtained from [here](../CreateOAuthState.md) <br/>
+`userCode`: (optional) A valid [User Code](../../../AccountService/Authentication/DeviceCode/Create.md) <br/>
+`continuation`: (optional) A valid Continuation Token <br/>
+`exchangeCode`: (optional), a valid exchange code (the usage is unknown) <br/>
+`codeChallenge`: (optional), a PKCE code challenge <br/>
+`codeChallengeMethod`: (optional), the method used to generate the code challenge (`S256` or `plain`) <br/>
+`nonce`: (optional) A valid [OpenID Nonce](https://curity.io/resources/learn/nonce-authenticator-pattern/#:~:text=The%20nonce%20authenticator%20pattern%20enables,session%20in%20the%20target%20application.&usg=AOvVaw2AvRUGFE9xV6e0C4-ifp_P&opi=89978449) <br/>
```
Comment (Jaren8r) [inline EpicGames/Web/Id/Auth/Redirect.md:18]:
```suggestion
`nonce`: (optional) A random value used to associate a request to a token. This value is reflected inside of epic_id tokens and id tokens. <br/>
```

**Review by Jaren8r — CHANGES_REQUESTED** [review]
There are a few more things. I do want to document the exchange code stuff but I'll do that in my own PR.

File: EpicGames/Web/Id/Auth/Redirect.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+## Id Web - Redirect
+
+URL: https://www.epicgames.com/id/api/redirect \
+Method: GET \
+Auth Required: Yes
+
+## Query Parameters
+
+`clientId`: The ID of the Client that the Authorization Code should be issued for <br/>
```
Comment (Jaren8r) [inline EpicGames/Web/Id/Auth/Redirect.md:9]:
```suggestion
`clientId`: The id of the client you are redirecting to <br/>
```

**Review by Jaren8r — APPROVED** [review]
Looks good, other than this which I should've caught earlier.

**Review by Jaren8r — APPROVED** [review]
Seems okay

**Pull request merged**